### PR TITLE
Added a note to the readme that Docker is required to run tests

### PR DESCRIPTION
**Pull request by emmberk**
and reformatted the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,30 +5,33 @@
...
-Boot up an [Apache Cassandra](https://cassandra.apache.org/) instance
-using the default hostname and ports: `localhost:9042`
-```
-./gradlew run
-```
-## Build Checks
-* ### OWASP check
-	`./gradlew dependencyCheckAnalyze --info`
+### Running Tests
+Tests are run automatically with `./gradlew build`. The tests require that Docker is running. Even if the tests fail, the artifacts are built and can be run.
```
Comment (Kjames5269) [inline README.md]: ✏️ Integration tests require...

Comment (emmberk) [reply README.md]: Thanks. I just confirmed that `./gradlew clean build -PskipITests` succeeds even if Docker isn't running.

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
CVE download failed, which shouldn't be affected by this PR. I'm overriding the CI build check.

**Pull request merged**